CMakeLists.txt

```diff
@@ -967,6 +976,10 @@ foreach(hal ${OpenCV_HAL})
     else()
       message(STATUS "Carotene: NEON is not available, disabling carotene...")
     endif()
+  elseif(hal STREQUAL "fastcv")
+    add_subdirectory(3rdparty/fastcv)
+    ocv_hal_register(FASTCV_HAL_LIBRARIES FASTCV_HAL_HEADERS FASTCV_HAL_INCLUDE_DIRS)
+    list(APPEND OpenCV_USED_HAL "fastcv (ver ${FASTCV_HAL_VERSION})")
```

Comment:
`FASTCV_HAL_VERSION` is not set on my machine

Reply:
we have set it in the fastcv CMakeList.txt file